test/spec/token.js

```diff
@@ -3419,3 +3421,109 @@ describe('token.verify', function() {
     });
   });
 });
+
+describe('token._addOAuthParamsToStorage', () => {
+  let sdkMock;
+  let setCookieMock;
+  let sessionStorageSetItemMock;
+  let hasSessionStorage;
```

Comment:
All local variables defined here should be reset in the `beforeEach` block. `hasSessionStorage` is not being cleared/reset in the `beforeEach` method, so the value from the previous test will remain unless it is set in the new test. This pattern can cause "flaky" tests since the behavior can vary based on what order the tests are run in, or if tests are run with other tests. But since this variable is a boolean it is difficult to reset.

In this case, I think it would be better to have a function like `mockSessionStorage(enabled: boolean)` which can be called by the test.